### PR TITLE
Changelog for 0.35 beta

### PR DESCRIPTION
**Pull request by jenshnielsen**

### CODE REVIEW

File: docs/changes/0.35.0beta1.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+QCoDeS 0.35.0beta1 (2022-07-08)
```
Comment (astafan8) [inline docs/changes/0.35.0beta1.rst:1]: beta1? not b1?

File: docs/changes/0.35.0beta1.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+===============================
+
+This is a beta release of the upcoming QCoDeS 0.35.0 release.
+
+In this release a large refactoring of the QCoDeS modules organization has taken place.
+Among other changes the `instrument` module has been split into a `parameters` and
+`instrument` module, yhe `dond` functions have been moved to the `dataset` module
```
Comment (astafan8) [inline docs/changes/0.35.0beta1.rst:8]:
```suggestion
`instrument` module, the `dond` functions have been moved to the `dataset` module
```

**Review by astafan8 — APPROVED** [review]
looks great!

File: docs/changes/0.35.0beta1.rst (hunk 1 above)
Comment (jenshnielsen) [reply docs/changes/0.35.0beta1.rst]: I was sure its beta but you are right (see pep440)

**Pull request merged**